trymito.io/styles/[slug].module.css

```diff
@@ -1,19 +1,102 @@
+.decorative_line {
+    position: absolute;
+    width: 1px;
+    height: calc(100% - 42rem);
+    border: 0.5px dashed rgba(208, 208, 208, 0.25);
 
+    /* 
+     * The decorative line is supposed to be aligned with the header and
+     * the content, and their width is dependent on the screen width. 
+     * Here's a breakdown of the calculation:
+        * 19.5rem: The width of the sidebar
+        * 100vw: The width of the screen
+        * 70rem: The width of the content
+        * 6px: The extra cushion so it doesn't touch the content
+     * The max function is used to keep the line from going too far to the left when the screen is narrow
+     * The width of the content is (100vw - 20rem) and it is centered, so the left margin 
+     */
+    left: max(19.5rem, calc(((100vw - 70rem) / 2) - 6px));
+}
+
+/* Post Styling */
 .post {
-    margin-top: 4rem;
+    margin-top: 7rem;
     margin-bottom: 4rem;
+    width: 100%;
+    font-family: Inter, Ubuntu, Helvetica, Arial, sans-serif;
+}
+
+.post a, .suggested_post h4 {
+    text-decoration: underline rgba(0, 0, 0, 0);
+    transition: text-decoration-color 300ms;
+}
+
+.post a:not(.suggested_post):hover, .suggested_post:hover h4 {
+    text-decoration-color: rgba(var(--color-purple), 1);
+}
+
+.post strong {
+    color: var(--color-text-primary);
+    font-weight: 500;
+}
+
+
+
+/* Post Banner */
+.post_banner {
+    background-color: var(--color-purple);
+    padding: 5rem 20rem;
+    width: 100%;
+    display: flex;
+    justify-content: center;
 }
 
 .post_title {
-    max-width: min(800px, 90vw);
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: column;
+    /* This is intentionally set to be the same width as the content. */
+    width: calc(100vw - 40rem);
+    max-width: 70rem;
 }
 
 .post_title h1 {
     font-size: 3rem;
 }
 
+.post_title p, .post_title h1 {
+    margin-top: 0
+}
+
+.post_metadata {
+    display: flex;
+    flex-direction: row;
+    margin-top: 2rem;
+}
+
+.post_additional_info {
+    margin-left: 3rem;
+}
+
+.blog_home_container {
+    margin-bottom: 2rem;
+}
+
+.blog_home_image_and_text {
+    display: flex;
+    flex-direction: row;
+    align-items: baseline;
+    cursor: pointer;
+}
+
+
+
+/* Post Content */
 .post_content {
-    max-width: min(720px, 90vw);
+    margin: 0 2rem;
+    max-width: 70rem;
+    /* Without this, there is a horizontal scrollbar on the page */
```

Comment:
This is helpful, but can you take it a step further and say why there is a horizontal scrollbar on the page?

Just based on my experience working on the website if we need to hardcode things like this to prevent a scrollbar from appearing, its often because we're hardcoding things elsewhere when they could otherwise be handled with a flex box that has proper margins and padding set. 

But I could be totally wrong!